### PR TITLE
Feat(Airdrop): Add STX airdrop completion info card

### PR DESCRIPTION
**Pull request by schnogz**
Screen Shot 2020-01-13 at 6 53 56 PM

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/Airdrops/StxAirdrop/index.tsx [hunk 1]
```diff
@@ -49,77 +53,130 @@ export type Props = {
...
-  return (
-    <Box>
-      <div>
-        <Header>
-          <Icon name='stx' color='stx' size='32px' />
+  const blockstackCampaign = props.userCampaignsInfoResponseList.find(
+    (campaign: CampaignInfoType) => campaign.campaignName === 'BLOCKSTACK'
+  )
+  return propEq('userCampaignState','NONE', blockstackCampaign) ? null : (
+    propEq('userCampaignState','REWARD_RECEIVED', blockstackCampaign) ? (
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Airdrops/StxAirdrop/index.tsx]: propEq is not type safe.

File: packages/blockchain-wallet-v4-frontend/src/scenes/Airdrops/StxAirdrop/index.tsx [hunk 2]
```diff
@@ -49,77 +53,130 @@ export type Props = {
...
+      <CompletedStxInfo />
+    ) : (
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Airdrops/StxAirdrop/index.tsx]: UserCampaignState =
  | 'FAILED'
  | 'REWARD_RECEIVED'
  | 'TASK_FINISHED'
  | 'REWARD_SEND'
  | 'REGISTERED'
  | 'NONE'

This means:
if userCampaignState is NONE show nothing
if userCampaignState is REWARD_RECEIVED show the new thing
if userCampaignState is FAILED, TASK_FINISHED, REGISTERED, or REWARD_SEND we show the old Box with StxStatus and that has calls to action. Are you sure we want that?

Comment (schnogz) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Airdrops/StxAirdrop/index.tsx]: hmm... yeah not sure about the last one

File: packages/blockchain-wallet-v4-frontend/src/scenes/Airdrops/StxAirdrop/index.tsx [hunk 3]
```diff
@@ -50,6 +28,12 @@ export type Props = {
 }
 
 const StxAirdrop = (props: Props & LinkDispatchPropsType) => {
+  const stxCampaign = props.userCampaignsInfoResponseList.find(
+    (campaign: CampaignInfoType) => campaign.campaignName === 'BLOCKSTACK'
+  )
+
+  if (!stxCampaign) return null
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Airdrops/StxAirdrop/index.tsx]: If we don't want to show to users that haven't participated we can add

if (stxCampagin.userCampaignState === 'NONE') return null

**Pull request merged**